Cart page implementation

srgzlb opened the pull request:
The purpose of this PR is to implement the Cart page

![Arc_2024-08-06 16-01-46@2x](https://github.com/user-attachments/assets/c871bfd5-c867-4de8-befb-e99b45528047)


![BQ Wedding Decor Cart](https://github.com/user-attachments/assets/f05f1531-b589-477d-b388-c88407019d0b)

dabrorius commented:
Can you double check how it looks for you? On my local machine these two buttons are no styled. Not sure if it's an issue with my environment.
![Screenshot 2024-08-06 at 13 52 57](https://github.com/user-attachments/assets/541280c6-de93-4843-9303-16880b0620da)

srgzlb commented:
> Can you double check how it looks for you? On my local machine these two buttons are no styled. Not sure if it's an issue with my environment. ![Screenshot 2024-08-06 at 13 52 57](https://private-user-images.githubusercontent.com/690113/355437520-541280c6-de93-4843-9303-16880b0620da.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI5NDgzNzAsIm5iZiI6MTcyMjk0ODA3MCwicGF0aCI6Ii82OTAxMTMvMzU1NDM3NTIwLTU0MTI4MGM2LWRlOTMtNDg0My05MzAzLTE2ODgwYjA2MjBkYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgwNlQxMjQxMTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01M2IyYzhmZjAzZDc2NjhmNjk1YTMyMTIxNWFmMjkxODgwNGViOWE2NWFhMTAyMDU5Zjc1YjkxYjMwOGU3OTFlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Td6vkIFwSs7WRaIXezAw3w-eOY9LakXR0QYoijc6oWE)

I've updated the PR's description and added some screenshots. It looks well on my production.
As far as I remember, during the development of Impact theme Jevin experienced the same issue with the cart page and its buttons when tested it on his local end
